File: src/test/java/at/chaoticbits/coinmarket/CoinMarketSchedulerTest.java
package at.chaoticbits.coinmarket;

import org.testng.Assert;
import org.testng.annotations.Test;


public class CoinMarketSchedulerTest {

    @Test
    public void testRun() {
        CoinMarketScheduler coinMarketScheduler = new CoinMarketScheduler();

        coinMarketScheduler.run();

        Assert.assertNotNull(CoinMarketContainer.erc20Tokens);
        Assert.assertNotNull(CoinMarketContainer.symbolSlugs);
    }
}

File: src/test/kotlin/at/chaoticbits/coinmarket/CoinMarketSchedulerTest.kt
package at.chaoticbits.coinmarket

import org.junit.Assert
import org.junit.Test


/**
 * Test CoinMarketCap Schedulers
 */
class CoinMarketSchedulerTest {

    @Test
    fun testRun() {

        val coinMarketScheduler = CoinMarketScheduler()

        coinMarketScheduler.run()

        Assert.assertNotNull(CoinMarketContainer.getErc20Tokens())
        Assert.assertNotNull(CoinMarketContainer.getCoins())
    }
}
